project.go

```diff
@@ -0,0 +1,392 @@
+	for _, e := range envs {
+		for _, name := range names {
+			if e.Name == name {
+				filtered = append(filtered, e)
+				break
+			}
+		}
+	}
+	return filtered
+}
+
+type commaSeparatedFlag struct {
+	values []string
+}
+
+func (f *commaSeparatedFlag) Values() []string {
+	return f.values
+}
+
+func (f *commaSeparatedFlag) String() string {
+	return strings.Join(f.values, ",")
+}
+
+func (f *commaSeparatedFlag) Set(v string) error {
+	f.values = strings.Split(v, ",")
+	return nil
+}
+
+func (f *commaSeparatedFlag) validate(validValues []string) error {
+	var invalidValues []string
+	for _, cv := range f.values {
+		var found bool
+		for _, validValue := range validValues {
+			if validValue == cv {
+				found = true
+				break
+			}
+		}
+		if !found {
+			invalidValues = append(invalidValues, cv)
+		}
+	}
+	if len(invalidValues) > 0 {
+		return fmt.Errorf("invalid values: %s (valid options are: %s)", strings.Join(invalidValues, ", "), strings.Join(validValues, ", "))
+	}
+	return nil
+}
+
+type projectRemove struct {
+	cmd.ConfirmationCommand
+	name string
+	fs   *gnuflag.FlagSet
+}
+
+func (c *projectRemove) Info() *cmd.Info {
+	return &cmd.Info{
+		Name: "project-remove",
+		Desc: "removes the given project",
+	}
+}
+
+func (c *projectRemove) Run(ctx *cmd.Context, client *cmd.Client) error {
+	if c.name == "" {
+		return errors.New("please provide the name of the project")
+	}
+	config, err := loadConfigFile()
+	if err != nil {
+		return errors.New("unable to load environments file, please make sure that tranor is properly configured")
+	}
+	if !c.Confirm(ctx, fmt.Sprintf("Are you sure you want to remove the project %q?", c.name)) {
+		return nil
+	}
+	envNames := config.envNames()
+	appNames := make([]string, len(envNames))
+	for i, envName := range envNames {
+		appNames[i] = fmt.Sprintf("%s-%s", c.name, envName)
+	}
+	errs, err := deleteApps(appNames, client)
+	if err != nil {
+		return err
+	}
+	var notFound int
+	for _, err := range errs {
+		if err != nil {
+			if e, ok := err.(*tsuruerrors.HTTP); ok && e.Code == http.StatusNotFound {
+				notFound++
+			}
+		}
+	}
+	if notFound == len(errs) {
+		return errors.New("project not found")
+	}
+	fmt.Fprintln(ctx.Stdout, "Project successfully removed!")
+	return nil
+}
+
+func (c *projectRemove) Flags() *gnuflag.FlagSet {
+	if c.fs == nil {
+		c.fs = c.ConfirmationCommand.Flags()
+		c.fs.StringVar(&c.name, "name", "", "name of the project to b remove")
+		c.fs.StringVar(&c.name, "n", "", "name of the project to remove")
+	}
+	return c.fs
+}
+
+type projectInfo struct {
+	name string
+	fs   *gnuflag.FlagSet
+}
+
+func (c *projectInfo) Info() *cmd.Info {
+	return &cmd.Info{
+		Name: "project-info",
+	}
+}
+
+func (c *projectInfo) Run(ctx *cmd.Context, client *cmd.Client) error {
+	if c.name == "" {
+		return errors.New("please provide the name of the project")
+	}
+	apps, err := c.projectApps(client)
+	if err != nil {
+		return err
+	}
+	fmt.Fprintf(ctx.Stdout, "Project name: %s\n\n", c.name)
```

Comment:
I suggest `project-list` contains additional platform + teams.

Just talked to Rafa and we propose this output for `project-info`
```
% tranor project-info -n website

Project name: website
Description:
Repository: git@git.tsuru.example.com:website.git
Platform: ruby
Teams: example-team
Owner: someone@example.com
Team owner: example-team


+-------------+-------------------------+-------+---------------+-------------------------------+-------+
| Environment | Address                 | Image | Git hash /tag |Deploy date                    | Units |
+-------------+-------------------------+-------+---------------+----------------+--------------+-------+
| dev         | website.dev.example.com | v2    | (git) 7d28fc  | Mon, 05 Sep 2016 16:36:19 UTC | 1     |
| prod        | website.example.com     | v1    | (git) 6e172a  | Tue, 06 Sep 2016 10:10:00 UTC | 2     |
+-------------+-------------------------+-------+---------------+-------------------------------+-------+
```

We also think it could be interesting for us to have a `project-env-info`, which would display the same as `app-info`:
```
% tranor project-env-info -n website -e dev

(same as app info)
```